.changeset/curvy-rabbits-run.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/control-property-editor': patch
+---
+
+"Confirm property change deletion"'s message on delete is only for property changes
```

Comment:
Please add patch for `preview-middleware`.

Reply:
Done

Reply:
Thx